bodhi/server/models.py

```diff
@@ -1248,6 +1305,43 @@ class Build(Base):
         'polymorphic_identity': ContentType.base,
     }
 
+    # Local cache in case we need more koji info
+    _kojiinfo = None
+
+    def _get_kojiinfo(self):
+        if not self._kojiinfo:
+            koji_session = buildsys.get_session()
+            self._kojiinfo = koji_session.getBuild(self.nvr)
+        return self._kojiinfo
+
+    def _get_n_v_r(self):
+        """Return the N, V and R components of an RPM build from NVR."""
+        return self.nvr.rsplit('-', 2)
+
+    @property
+    def nvr_name(self):
+        """Return the RPM Name."""
+        return self._get_n_v_r()[0]
+
+    @property
+    def nvr_version(self):
+        """Return the RPM Version."""
+        return self._get_n_v_r()[1]
+
+    @property
+    def nvr_release(self):
+        """Return the RPM Release."""
+        return self._get_n_v_r()[2]
+
+    def get_n_v_r(self):
+        """
+        Return the (name, version, release) of this build.
+
```

Comment:
Maybe let's add this as a second sentence in this docblock:

Note: This does not directly return the Build's nvr attribute, which is a str.